app/controllers/AddAdditionalMetadataController.scala

```diff
@@ -34,248 +34,104 @@ class AddAdditionalMetadataController @Inject() (
 )(implicit val ec: ExecutionContext)
     extends TokenSecurity
     with I18nSupport {
-  private val mainFormPageTitle = "Add %s metadata to"
-  private val mainFormPageDescription = "Enter metadata for %s fields here."
-  private val dependencyFormPageTitle = "Add %s to"
-  private val dependencyFormPageDescription =
-    s"""Enter a publicly visible %s if, for example, %s sensitive information.
-       | For guidance on how to create %s, read our FAQs (opens in a new tab)""".stripMargin
 
-  def addAdditionalMetadata(propertyNameAndFieldSelected: List[String], consignmentId: UUID, metadataType: String, fileIds: List[UUID]): Action[AnyContent] =
-    standardTypeAction(consignmentId) { implicit request: Request[AnyContent] =>
+  def addAdditionalMetadata(consignmentId: UUID, metadataType: String, fileIds: List[UUID]): Action[AnyContent] = standardTypeAction(consignmentId) {
+    implicit request: Request[AnyContent] =>
       for {
-        formData <- getDefaultFieldsForForm(metadataType, isMainForm = true, convertNameAndFieldToObject(propertyNameAndFieldSelected), consignmentId, request)
-        (pageInfo, controllerInfo) <- getInfoForAddAdditionalMetadataPage(
-          consignmentId,
-          request,
-          metadataType,
-          isMainForm = true,
-          propertyNameAndFieldSelected,
-          formData.formFields,
-          fileIds
-        )
+        consignment <- consignmentService.getConsignmentFileMetadata(consignmentId, request.token.bearerAccessToken, Option(FileFilters(None, Option(fileIds), None, None)))
+        defaultFieldForm <- getDefaultFieldsForForm(consignmentId, request)
```

Comment:
Not sure this should be called `defaultFieldForm`, maybe better to call it: `formFields`, as it is only a single page, don't think these are "default" fields any more

Reply:
Yeah, that makes sense.